designsystem/src/commonMain/kotlin/org/hisp/dhis/mobile/ui/designsystem/resource/Image.kt

```diff
@@ -13,3 +13,8 @@ import org.jetbrains.compose.resources.painterResource
 @Composable
 fun provideDHIS2Icon(resourceName: String): Painter =
     painterResource("drawable/$resourceName.xml")
+
+@OptIn(ExperimentalResourceApi::class)
+@Composable
+fun provideImage(resourceName: String): Painter =
```

Comment:
I thin that this method can be set in the `common` module the library is not intended to store jpgs, right?

Reply:
It is just for the purpose of adding a painter to the Image and as a sample, it is in the same place as the providedhis2Icon method so I think it's ok, it's not intended to store jpgs and the user won't use it, as if they pass the painter with their own resource it'll work on both platforms